packages/sirius-web/backend/sirius-web-application/src/main/resources/schema/libraries.graphqls

```diff
@@ -13,6 +13,7 @@ type ViewerLibrariesEdge {
 }
 
 type Library {
+  id: ID!
```

Comment:
I put back the ID to ease my life with the library selection table.
If necessary I can remove it and use the `namespace:name:version` instead

Reply:
I haven't reviewed the whole thing but I've just stumble upon this comment. Prefer the namespace, name and version indeed. I'm not fan of exposing the concept of libraryId in our APIs, this is not how people should think about the identity of a library.

Reply:
Let's keep that for now if it creates some additional complexity. I'll see later what we can do on that topic

Reply:
It requires a bit of work on the table in the frontend to merge the `namespace:name:version` into a valid Material React Table row ID, and split it before sending the GraphQL query to the backend.
Since the mutation should allow to import multiple libraries, its input should provide a list of (`namespace`, `name`, `version`). I don't know GraphQL enough to know the best way to do that, it could maybe be something like
```
input PublishLibrariesInput {
    libraries: [LibraryMetadata!]!
}

type LibraryMetadata {
    namespace: String!
    name: String!
    version: String!
}
```

Reply:
I suspect that in the end, the id of a library could literally become `namespace:name:version` since none of those property is mutable.